src/components/Image/Image.tsx

```diff
@@ -6,8 +6,10 @@ import ImageElement from './ImageElement';
 type ImageProps = {
   alt: string;
   caption?: string;
+  captionVariant?: 'below' | 'right';
   credit?: string;
   className?: string;
+  hideCaption?: boolean;
```

Comment:
To align with other boolean prop names, I think we should use the same convention. So this could be `shouldHideCaption` instead.

I just looked this up, and apparently these words are called "modal auxilliary verbs" in English grammar 😎 and there are 9 of them: shall, should, can, could, will, would, may, must, might. We should always use these when naming boolean props.

https://www.google.com/search?q=modal+auxiliary+verb